[34m[1mwandb[39m[22m: [32m[41mERROR[39m[49m Project chweber/output does not contain artifact: "dv.pkl"
Traceback (most recent call last):
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/apis/public.py", line 928, in artifact
    artifact = artifacts.PublicArtifact(self.client, entity, project, artifact_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/artifacts/public_artifact.py", line 225, in __init__
    self._attrs = attrs or self._load()
                           ^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/artifacts/public_artifact.py", line 998, in _load
    raise ValueError(
ValueError: Project chweber/output does not contain artifact: "dv.pkl"
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/train.py", line 66, in <module>
    run_train()
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/toph/Documents/Private/GitHub/MLOps-Zoomcamp/train.py", line 41, in run_train
    artifact = wandb.use_artifact(data_artifact, type="preprocessed_dataset")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 342, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 332, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 2675, in use_artifact
    artifact = public_api.artifact(type=type, name=name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/apis/normalize.py", line 87, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/apis/public.py", line 928, in artifact
    artifact = artifacts.PublicArtifact(self.client, entity, project, artifact_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/artifacts/public_artifact.py", line 225, in __init__
    self._attrs = attrs or self._load()
                           ^^^^^^^^^^^^
  File "/opt/homebrew/anaconda3/envs/mlops_zoomcamp/lib/python3.11/site-packages/wandb/sdk/artifacts/public_artifact.py", line 998, in _load
    raise ValueError(
wandb.errors.CommError: Project chweber/output does not contain artifact: "dv.pkl"